x-pack/test/security_solution_endpoint/apps/integrations/artifact_entries_list.ts

```diff
@@ -51,10 +51,13 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       .delete(`${EXCEPTION_LIST_URL}?list_id=${listId}&namespace_type=agnostic`)
       .set('kbn-xsrf', 'true');
   };
-
+  // It's flaky only in Serverless
   // Failing: See https://github.com/elastic/kibana/issues/187314
   // Failing: See https://github.com/elastic/kibana/issues/187383
-  describe.skip('@ess @serverless For each artifact list under management', function () {
+  describe('For each artifact list under management', function () {
+    targetTags(this, ['@ess', '@skipInServerless']);
```

Comment:
Question: is targetTags something we are switching to from inline tags?

Reply:
security_solution_api_integration supports inline tags; while  `x-pack/test` supports only `targetTags`.
Given that we are moving tests back to `x-pack/test`, we should revert the way we apply tags as well.